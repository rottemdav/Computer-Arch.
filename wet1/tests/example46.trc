1 8 18 3 global_history local_tables using_share_mid
0x6a7e8 T 0x401b0f4
0x6dd18 N 0x0e8
0xce797d0 N 0x350
0xecca00 N 0xa9b0
0x7922124 N 0x9490
0x917eb8 N 0xf768
0x57e2ef0 N 0x706c4
0x917eb8 T 0xf768
0x25f74 N 0xd63ef384
0xb88 N 0x4b11763c
0xdd4 T 0x3fc50
0x6a7e8 N 0x401b0f4
0xbc8 N 0xbb70
0x630 N 0x310
0x7922124 N 0x9490
0x91594 N 0xfc7cde4
0x688fb60c T 0xd9b725c
0x7642e000 T 0x9a172e78
0x7642e000 T 0x9a172e78
0x2b10 N 0x24e94
0x66a6a0 T 0x4cc
0x77b4 N 0xbb9fa778
0x89d0e1b8 T 0xe1e34
0x134d014 T 0x570910
0x2b10 T 0x24e94
0x7642e000 T 0x9a172e78
0xecca00 N 0xa9b0
0xe7f4 T 0x7d78c
0x301e89fc N 0x5d213ed8
0x91594 T 0xfc7cde4
0xecca00 N 0xa9b0
0x484 T 0xb2a4
0x484 T 0xb2a4
0x91594 T 0xfc7cde4
0x6dd18 T 0x0e8
0xdd4 T 0x3fc50
0x2a2202c N 0xec5f20
0x484 T 0xb2a4
0x2869de8 T 0xe6a85c38
0x48a908 T 0x290
0x91594 T 0xfc7cde4
0x25f74 N 0xd63ef384
0x630 T 0x310
0x484 T 0xb2a4