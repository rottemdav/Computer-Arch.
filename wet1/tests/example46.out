0x6a7e8 N 0x6a7ec
0x6dd18 N 0x6dd1c
0xce797d0 N 0xce797d4
0xecca00 N 0xecca04
0x7922124 N 0x7922128
0x917eb8 N 0x917ebc
0x57e2ef0 N 0x57e2ef4
0x917eb8 N 0x917ebc
0x25f74 N 0x25f78
0xb88 N 0xb8c
0xdd4 N 0xdd8
0x6a7e8 N 0x6a7ec
0xbc8 N 0xbcc
0x630 N 0x634
0x7922124 N 0x7922128
0x91594 N 0x91598
0x688fb60c N 0x688fb610
0x7642e000 N 0x7642e004
0x7642e000 T 0x9a172e78
0x2b10 N 0x2b14
0x66a6a0 N 0x66a6a4
0x77b4 N 0x77b8
0x89d0e1b8 N 0x89d0e1bc
0x134d014 N 0x134d018
0x2b10 N 0x2b14
0x7642e000 N 0x7642e004
0xecca00 N 0xecca04
0xe7f4 N 0xe7f8
0x301e89fc N 0x301e8a00
0x91594 N 0x91598
0xecca00 N 0xecca04
0x484 N 0x488
0x484 T 0xb2a4
0x91594 N 0x91598
0x6dd18 N 0x6dd1c
0xdd4 N 0xdd8
0x2a2202c N 0x2a22030
0x484 N 0x488
0x2869de8 N 0x2869dec
0x48a908 N 0x48a90c
0x91594 N 0x91598
0x25f74 N 0x25f78
0x630 N 0x634
0x484 N 0x488
flush_num: 22, br_num: 44, size: 569b
